### ISSUE TITLE
action amondnet/now-deployment@v2 often fails

### ISSUE THREAD

**Issue by tpltnt** [flattenthecurve/guide#135]
Hi there,

a lot of PRs are blocked due to `amondnet/now-deployment@v2` failing. The error is always
```
Now CLI 17.0.4
Error! No existing credentials found. Please run "now login" or pass "--token"
> More details: https://err.sh/now/no-credentials-found
```
Examples are #132 #129 #125 
This blocks useful PRs and puts unnecessary load on the people who try to help (only due to an avoidable technical issue). Is the deployment action actually needed? (And would a test for building the site suffice?)

Cheers,
tpltnt

**Comment by matiasgarciaisaia** [comment]
@bcardiff any ideas on this?

**Comment by bcardiff** [comment]
Two issues:

* The zeit-team-id is missing, that is fixed on 7cfc4cbdcdd7e9ef75bab1e5a185fb4b04ec70ab but is not in the base of the some of those PR.

* There has been more request to the preview request than we are allowed in the current tier. Will need to wait, upgrade plan or decouple the CI.

I understand is noisy, but it does not block merging some PRs that are non-risky content changes.

**Comment by matiasgarciaisaia** [comment]
zeit.co has improved our usage limit in exchange for [a Tweet](https://twitter.com/flattencurve/status/1240727894936236038) and [some attribution on the site](https://github.com/flattenthecurve/guide/commit/2ec308e5558bb2f2be19c4ec36dc39f95988a104). We should not hit the rate limit anymore.
